### PR TITLE
compare Interpro domains of two systems 

### PR DESCRIPTION
**Pull request by leetina4**
…unction

### CODE REVIEW

File: R/symComp.R [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        d <- c(d, names(s2[k]))
+      }
+    }
+    if (length(d) > 0) {
+      dom[[i]] <- d
+    }
+  }
+  names(dom) <- tmp
+
+  # Plot horixontal graph of results
```
Comment (rn1n3r) [inline R/symComp.R]: typo: horizontal

File: R/symComp.R [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    for (j in seq_along(s2)) {
+      ints <- unlist(intersect(s1[[i]], s2[[j]]))
+      if (length(ints > 0)) {
+        tmp <- unique(c(tmp, ints))
+      }
+    }
+  }
+
+  dom <- list()
+  for (i in seq_along(tmp)) {
```
Comment (rn1n3r) [inline R/symComp.R]: Perhaps a comment or two here would help explain what this loop is doing

File: R/symComp.R [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+#' @author {Tina Lee} (aut)
+#' @examples
+#' # Picking sample system1 and system2 to get intersect domains of the
+#' # two systems
+#' # Call the symComp helper function to generate list
+#' result <- symComp("PHALY", "SLIGR")
+#'
+#' @export
+
+symComp <- function(system1, system2) {
```
Comment (rn1n3r) [inline R/symComp.R]: Perhaps consider naming the function "sysComp" to make it more clear that it's comparing two systems?

Comment (rn1n3r) [inline R/symComp.R]: Perhaps consider naming the function "sysComp" to make it more clear that it's comparing two systems?

File: R/symComp.R (hunk 2 above)
Comment (rn1n3r) [inline R/symComp.R]: Perhaps a comment or two here would help explain what this loop is doing

File: R/symComp.R (hunk 1 above)
Comment (rn1n3r) [inline R/symComp.R]: typo: horizontal

**Review by rn1n3r — CHANGES_REQUESTED** [review]

File: R/symComp.R [hunk 4]
```diff
@@ -0,0 +1,89 @@
+# symComp.R
+
+#'
+#' \code{symComp} Return a list of domains that the genes in two systems
+#'    overlap along with genes that mapped to the same domains
+#'
+#' @param system1 The name of the system in interest. Is compared with system2
```
Comment (sapirl) [inline R/symComp.R]: The @param and @return need to be changed according to the format described here:
https://github.com/hyginn/BCB420.2019.ESA/blob/master/PullRequestReviewNotes.md

The format is: mention the type and dimension of parameters and return values in the header. Type is one of {numeric|integer|double|complex|logical|character|function} ... or a mixed type such as {data frame|list}. The dimension is one of {a|a vector|a matrix|a 3D matrix|...},
For example:
@return (numeric)  A vector of the next six 6-of-49 lottery numbers or NULL.

**Comment by hyginn** [comment]
Conflict needs to be resolved so Travis can check the package!

**Review by hyginn — COMMENTED** [review]
Looks oK - but can't really assess without documentation, example runs and interpretation. No Vignette.

**Comment by hyginn** [comment]
Evaluated as-is. Won't merge. Closed.

**Pull request closed without merging**